src/components/vm/disks/diskAdd.jsx

```diff
@@ -583,6 +584,23 @@ export class AddDiskModalBody extends React.Component {
             storagePool = storagePools.find(pool => pool.name === this.state.storagePoolName);
             volume = storagePool.volumes.find(vol => vol.name === this.state.existingVolumeName);
             isVolumeUsed = getStorageVolumesUsage(vms, storagePool);
+        } else if (this.state.mode === CUSTOM_PATH) {
+            // fallback to raw.
+            format = "raw";
```

Comment:
Let's move this to the `.catch()`, to avoid race conditions and make the code a bit cleaner.